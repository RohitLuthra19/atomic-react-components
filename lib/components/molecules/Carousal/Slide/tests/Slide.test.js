import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';

import { SlideVanilla as Slide } from '../index';

describe('<Slide />', () => {
  let SlideComponent = '';
  beforeEach(() => {
    SlideComponent = shallow(<Slide className="test">Test</Slide>);
  });

  test('should render correctly', () => {
    expect(SlideComponent).toMatchSnapshot();
  });
});

describe('Slide Component', () => {
  let SlideComponent = '';
  beforeEach(() => {
    SlideComponent = shallow(<Slide className="test">Test</Slide>);
  });

  test('should have active-slide class if isActive prop is true', () => {
    SlideComponent.setProps({
      isActive: true,
    });
    expect(SlideComponent.find('div').hasClass('active-slide')).toBeTruthy();
  });
});
