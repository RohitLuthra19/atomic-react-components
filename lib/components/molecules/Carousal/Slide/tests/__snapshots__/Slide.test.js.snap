// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slide /> should render correctly 1`] = `
<div
  aria-hidden={true}
  className="test"
  onClick={[Function]}
  onKeyPress={[Function]}
  role="button"
  tabIndex="-1"
>
  Test
</div>
`;
